### ISSUE TITLE
/mooculus/review/refreshRationalFunctions/exerciseList/review/refreshRationalFunctions/longDivision

### ISSUE THREAD

**Issue by cwiswesser37** [XimeraProject/server#329]
The page isn't loading correctly. None of the problems are being displayed.

**Comment by cwiswesser37** [comment]
This problem is still persisting. [https://imgur.com/a/1z7GMVf](url)

**Comment by bartsnapp** [comment]
Ok, I just checked it out on firefox...

![image](https://user-images.githubusercontent.com/1376703/45719495-89f2fd80-bb6e-11e8-8652-bada3a09533f.png)

I checked out your imgur link... I think you are running chrome. 

Do other pages load? 

If other pages load, try this:

do a hard reload (usually shift-cntr R works) 

sometimes the math (powered by mathjax  https://www.mathjax.org/
takes a while to load. 

However, say 2 minutes should be enough....
